spec/controllers/hq/orphans_controller_spec.rb

```diff
@@ -10,8 +10,65 @@
 
   specify '#show' do
     expect(Orphan).to receive(:find).and_return(orphan)
+
     get :show, id: orphan.id
+
     expect(assigns(:orphan)).to eq orphan
     expect(response).to render_template 'show'
   end
+
+  context 'new and #create' do
+    let(:orphan) { build_stubbed :orphan }
+
+    before :each do
+      controller.instance_variable_set(:@orphan, orphan)
```

Comment:
Moot point since this code is slated for deletion, but I don't think it's a good idea to override a key function of the controller under test.